src/main/webapp/scripts/sessioncache.js

```diff
@@ -1,12 +1,14 @@
+import { Poller } from './poller';
+import { Session } from './session';
+
 /** 
  * SessionCache bridges the gap between the client and server.
  * Allows client to indirectly stay in contact with the server,
  * encapsulating many of the contact points away from the client.
  * 
  * SessionCache looks for information about the current session and
  * updates the server with information about each client in return (like
- * date last contacted). Also includes some additional functionality, 
- * like contacting the server to update the controller of the session.
+ * date last contacted).
```

Comment:
The cache doesn't do anything to the server. This is not accurate. All the cache does is ask the server for information. It, itself, does not invoke the the server to do anything. Please rephrase this such that it doesnt mention anything about the cache updating the server. That isnt accurate. This is a pretty important distinction. Let me know if it doesnt make sense

Reply:
I think I am still confused on where changing timeLastPolled comes into play here. I thought it would be done at the same time as when the session entity is retrieved.

Reply:
I think this a technicality. IIUC we are implicitly telling the server to update the timeLastPolled, but this should not cause the server to immediately take action based on that info. The background task should do that.

Reply:
ahh that makes a lot of sense thank you!